frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -295,3 +295,51 @@ Feature: Pipeline Runs
               And user clicks on Start
              Then user will be redirected to Pipeline Run Details page
               And user will see PVC Workspace "workspace-ex" mentioned in the VolumeClaimTemplate Resources section of Pipeline Run Details page
+
+
+        @regression, @manual
+        Scenario: Pipeline Run details page with passed pipeline run having finally task
+            Given pipeline with finally task having a passed pipeline run is present
```

Comment:
Given pipeline run is displayed for "pipeline-one" with finally task something like this might make more sense

Reply:
Updated